addon/mixins/serializer.js

```diff
@@ -1,52 +1,31 @@
 import Ember from 'ember';
```

Comment:
What about calling this file `addon/mixins/base_serializer.js` instead of `addon/mixins/serializer.js`?